dashboard/app/views/courses/index.html.haml

```diff
@@ -23,5 +23,9 @@
   = render file: Rails.root.join('..', 'shared', 'haml', 'course_explorer_table.haml')
   = render file: Rails.root.join('..', 'shared', 'haml', 'tool_explorer_table.haml')
   = render partial: 'home/all_courses'
+  -if current_user
+    - unless current_user.teacher?
+      #user_hero
+        = render template: 'api/user_hero'
```

Comment:
Is it bad if we skip these checks to reduce the amount of ruby code, and just rely on the React code to show it or not?